Lesson_12/Tusk_2.py

```diff
@@ -0,0 +1,23 @@
+from time import ctime
+
+
+def func_decorator(func):
+    def inside_func(*args, **kwargs):
+        func(*args, **kwargs)
```

Comment:
Assign the result of the `func` call to some variable and return this variable after writing to a file, but outside the context manager block. It will allow you to decorate functions that return some value